src/Azure.Functions.Cli/Interfaces/IDurableManager.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Threading.Tasks;
+
+namespace Azure.Functions.Cli.Interfaces
+{
+    internal interface IDurableManager
+    {
+        Task DeleteHistory();
```

Comment:
`DeleteHistory()` should be renamed to `DeleteTaskHub()`.

Reply:
Separately, let's add a new command called `PurgeHistory` which internally calls `PurgeOrchestrationHistoryAsync` (new in v1.3.2).